.tekton/pipeline.yaml

```diff
@@ -42,45 +42,45 @@ spec:
       workspaces:
         - name: task-pvc
           workspace: python-tracer-ci-pipeline-pvc
-    - name: unittest-cassandra
-      runAfter:
-        - clone
-      matrix:
-        params:
-          - name: imageDigest
-            value:
-              # 3.9.19-bookworm
-              - "sha256:e298e2e898691a938073f670dac8ef1a551c83344b67b5d8e32d1fbc8e0b57f8"
-      taskRef:
-        name: python-tracer-unittest-cassandra-task
-      workspaces:
-        - name: task-pvc
-          workspace: python-tracer-ci-pipeline-pvc
-    - name: unittest-couchbase
-      runAfter:
-        - clone
-      matrix:
-        params:
-          - name: imageDigest
-            value:
-              # 3.9.19-bookworm
-              - "sha256:e298e2e898691a938073f670dac8ef1a551c83344b67b5d8e32d1fbc8e0b57f8"
-      taskRef:
-        name: python-tracer-unittest-couchbase-task
-      workspaces:
-        - name: task-pvc
-          workspace: python-tracer-ci-pipeline-pvc
-    - name: unittest-gevent-starlette
-      runAfter:
-        - clone
-      matrix:
-        params:
-          - name: imageDigest
-            value:
-              # 3.9.18-bookworm
-              - "sha256:530d4ba717be787c0e2d011aa107edac6d721f8c06fe6d44708d4aa5e9bc5ec9"
-      taskRef:
-        name: python-tracer-unittest-gevent-starlette-task
-      workspaces:
-        - name: task-pvc
-          workspace: python-tracer-ci-pipeline-pvc
+    # - name: unittest-cassandra
+    #   runAfter:
+    #     - clone
+    #   matrix:
+    #     params:
+    #       - name: imageDigest
+    #         value:
+    #           # 3.9.19-bookworm
+    #           - "sha256:e298e2e898691a938073f670dac8ef1a551c83344b67b5d8e32d1fbc8e0b57f8"
+    #   taskRef:
+    #     name: python-tracer-unittest-cassandra-task
+    #   workspaces:
+    #     - name: task-pvc
+    #       workspace: python-tracer-ci-pipeline-pvc
+    # - name: unittest-couchbase
+    #   runAfter:
+    #     - clone
+    #   matrix:
+    #     params:
+    #       - name: imageDigest
+    #         value:
+    #           # 3.9.19-bookworm
+    #           - "sha256:e298e2e898691a938073f670dac8ef1a551c83344b67b5d8e32d1fbc8e0b57f8"
+    #   taskRef:
+    #     name: python-tracer-unittest-couchbase-task
+    #   workspaces:
+    #     - name: task-pvc
+    #       workspace: python-tracer-ci-pipeline-pvc
+    # - name: unittest-gevent-starlette
+    #   runAfter:
+    #     - clone
+    #   matrix:
+    #     params:
+    #       - name: imageDigest
+    #         value:
+    #           # 3.9.18-bookworm
```

Comment:
Same as [this](https://github.com/instana/python-sensor/pull/571/files#r1698713275)